docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -40,6 +40,34 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     connector.name=iceberg
     hive.metastore.uri=thrift://localhost:9083
 
+.. list-table:: Iceberg configuration properties
+  :widths: 35, 80, 5
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default
+  * - ``iceberg.file-format``
+    - Define the data storage file format for Iceberg tables.
+      Possible values are
+      
+      * ``PARQUET``
+      * ``ORC``
+    - ``ORC``
+  * - ``iceberg.compression-codec``
+    - The compression codec to be used when working with files. 
```

Comment:
this is purely relevant for writing, I'd revert to that wording.

Reply:
Nice catch, somewhere when rewriting, I must have rewritten this. Done